DRAFT_RELEASE_NOTES.md

```diff
@@ -75,3 +75,7 @@ To start your aiSSEMBLE upgrade, update your project's pom.xml to use the 1.9.0
 1. Repeat the previous step until all manual actions are resolved
 
 # What's Changed
```

Comment:
A: Let's focus on what we upgraded and where we upgraded from. So what was the MLFlow version change, what was the MLFlow chart version change, what was the base MLFlow image version change.

Reply:
Revised the section.